src/components/Navbar.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { HomeIcon } from '@heroicons/react/24/solid';
+import { User } from './User';
+
+const Navbar = () => (
```

Comment:
Can we add a story for the new `<Navbar>`? And/or for the `<Header>`?

I'm not sure how the `<User>` component will work, though; I'll have to look at the Storybook docs...

Reply:
I wasn't thinking that we would bother making stories for "one-time use" components… but as an artifact of communicating use, maybe it's worthwhile even for that. Good idea.